### PR TITLE
HAI-1523 Prevent deletion of hanke if it has applications being handled in Allu

### PR DESCRIPTION
**Pull request by markohaarni**
# Description

User can not delete hanke if it has any applications that have proceeded to being handled in Allu. If it has such applications cancel button is hidden in hanke view. In a case where user has pressed the cancel button when at that moment no application is being handled but that changes before user confirms cancelling, info dialog is displayed to user that deletion is not possible.

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Other

# Checklist:

- [x] I have written new tests (if applicable)
- [x] I have ran the tests myself (if applicable)
- [ ] I have made necessary changes to the documentation, link to confluence
      or other location:

### CODE REVIEW

File: src/locales/sv.json [hunk 1]
```diff
@@ -756,4 +751,4 @@
       "linkText": "SV: Avaa rekisteriseloste tästä (pdf)"
     }
   }
-}
+}
```
Comment (pitkni) [inline src/locales/sv.json:754]: A line break should be added to end of file.

File: src/locales/en.json [hunk 2]
```diff
@@ -756,4 +751,4 @@
       "linkText": "EN: Avaa rekisteriseloste tästä (pdf)"
     }
   }
-}
+}
```
Comment (pitkni) [inline src/locales/en.json:754]: A line break should be added to end of file.

File: src/locales/sv.json (hunk 1 above)
Comment (markohaarni) [reply src/locales/sv.json]: Line break added.

File: src/locales/en.json (hunk 2 above)
Comment (markohaarni) [reply src/locales/en.json]: Line break added.

**Review by pitkni — APPROVED** [review]

**Pull request merged**